docs/content/querying/segmentmetadataquery.md

```diff
@@ -139,6 +140,11 @@ null if the aggregators are unknown or unmergeable (if merging is enabled).
 
 * The form of the result is a map of column name to aggregator.
 
+#### rollup
+
+* `rollup` in the result is true/false/null.
```

Comment:
there is a case, if previous segment is rolled up, and the next segment use new spec which is not to rollup, then for segmentMetadataQuery should handle this case and return null if merge=true
Otherwise, rollup should be always true or false